addon_imps/storage/bitbucket.py

```diff
@@ -32,13 +32,43 @@ def first_cursor_str(self) -> str:
 @dataclasses.dataclass
 class BitbucketStorageImp(storage.StorageAddonHttpRequestorImp):
     async def get_external_account_id(self, auth_result_extras: dict[str, str]) -> str:
+        self.token = auth_result_extras.get("token")
```

Comment:
I don't see the necessity to assign this attribute, moreover this class would be destroyed after response is returned